src/views/Notifications/Home/_components/_utils.tsx

```diff
@@ -104,39 +104,42 @@ export const ActivityCTA = ({
   return <></>;
 };
 
-export const pushValuesToChangesArray = (
-  obj: ChangeTypes,
-  changesArray: string[]
-) => {
-  Object.entries(obj).forEach(([key, value]) => {
-    if (key !== '__typename') {
-      // Ignore key values that where the key is '__typename'
-      if (Array.isArray(value) && value.length > 0) {
-        // Ignore empty arrays (i.e. [])
-        if (key === 'added') {
-          // Model Lead added is important, so pushed to the front of array to ensure it will always be shown
-          changesArray.unshift(key);
-        } else {
+export const pushValuesToChangesArray = (obj: ChangeTypes): string[] => {
+  const changesArray: string[] = [];
+
+  const pushValues = (innerObj: ChangeTypes) => {
+    Object.entries(innerObj).forEach(([key, value]) => {
+      if (key !== '__typename') {
+        // Ignore key values that where the key is '__typename'
+        if (Array.isArray(value) && value.length > 0) {
+          // Ignore empty arrays (i.e. [])
+          if (key === 'added') {
+            // Model Lead added is important, so pushed to the front of array to ensure it will always be shown
+            changesArray.unshift(key);
+          } else {
+            changesArray.push(key);
+          }
+        } else if (typeof value === 'string' && value.trim() !== '') {
+          // Push the key if only the value is not an empty string (i.e. '')
           changesArray.push(key);
-        }
-      } else if (typeof value === 'string' && value.trim() !== '') {
-        // Push the key if only the value is not an empty string (i.e. '')
-        changesArray.push(key);
-      } else if (typeof value === 'number') {
-        // Push the key if the value is a number
-        changesArray.push(key);
-      } else if (typeof value === 'boolean' && value) {
-        // Push the key if the boolean value is true, ignore if false
-        changesArray.push(key);
-      } else if (value !== null && typeof value === 'object') {
-        if (key === 'crTdls' || key === 'planDiscussions') {
-          // Push the key if it is 'crTdls' & 'planDiscussions'
+        } else if (typeof value === 'number') {
+          // Push the key if the value is a number
+          changesArray.push(key);
```

Comment:
I think some of these lines are self-explanatory and dont need comments.  Really just looking for clarity as a whole , or where code may be a little complex